.github/workflows/docker.yaml

```diff
@@ -1,38 +1,60 @@
-        service_account_email: ${{ secrets.GCP_SA_EMAIL }}
-        service_account_key: ${{ secrets.GCLOUD_AUTH }}
-    - run: |
-        gcloud auth configure-docker
+    - uses: actions/checkout@v2
+    - name: Install AWS CLI
+      run: |
+        curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+        unzip -q awscliv2.zip
+        sudo ./aws/install
+        aws --version
+    - name: Install kubectl
+      run: |
+        curl -LO https://storage.googleapis.com/kubernetes-release/release/v$KUBECTL_VERSION/bin/linux/amd64/kubectl
+        chmod +x ./kubectl
+        sudo mv ./kubectl /usr/local/bin/kubectl
+        kubectl version --client
+        mkdir -p $HOME/.kube
+    - name: AWS auth with ECR
+      env:
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ECR_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_ECR_SECRET_ACCESS_KEY }}
+      run: |
+        aws ecr-public get-login-password --region us-east-1 > /tmp/aws
+        cat /tmp/aws | docker login --username AWS --password-stdin $DOCKER_ORG
+        rm /tmp/aws
     - name: Build
-      run: |        
-        docker build -t gcr.io/cdssnc/notify/document-download-api:${GITHUB_SHA::7} -t gcr.io/cdssnc/notify/document-download-api:latest -f ci/Dockerfile .
+      run: |
+        docker build --build-arg GIT_SHA=${GITHUB_SHA::7} -t $DOCKER_SLUG:${GITHUB_SHA::7} -t $DOCKER_SLUG:latest -f ci/Dockerfile .
     - name: Publish
       run: |
-        docker push gcr.io/cdssnc/notify/document-download-api:latest && docker push gcr.io/cdssnc/notify/document-download-api:${GITHUB_SHA::7}
-    - name: EKS
-      uses: docker://gcr.io/cdssnc/aws:latest
+        docker push $DOCKER_SLUG:latest && docker push $DOCKER_SLUG:${GITHUB_SHA::7}
+    - name: Get Kubernetes configuration
       env:
         AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
         AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-      with:
-        args: eks --region ca-central-1 update-kubeconfig --name notification-canada-ca-staging-eks-cluster
-    - name: Apply
-      uses: docker://gcr.io/cdssnc/aws-kubectl:latest
+      run: |
+        aws eks --region $AWS_REGION update-kubeconfig --name notification-canada-ca-staging-eks-cluster --kubeconfig $HOME/.kube/config
+    - name: Update image in staging
       env:
         AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
         AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-      with:
-        args: set image deployment.apps/document-download-api document-download-api=gcr.io/cdssnc/notify/document-download-api:${GITHUB_SHA::7} -n=notification-canada-ca --kubeconfig=/github/home/.kube/config
+      run: |
+        kubectl set image deployment.apps/document-download-api document-download-api=$DOCKER_SLUG:${GITHUB_SHA::7} -n=notification-canada-ca --kubeconfig=$HOME/.kube/config
+    - uses: cds-snc/notification-pr-bot@master
+      env:
+        TOKEN: ${{ secrets.PR_BOT_TOKEN }}
```

Comment:
Is the PR bot a new addition here?

Reply:
It is, thanks for being careful! The PR bot is not hooked up yet, so now it'll do nothing but this change is required if we want to have document download API + frontend changes in the automated PR